### DISCUSSION TITLE
MAUIsland partnership?

### DISCUSSION THREAD

**Discussion by Strypper** [yiszza/Material.Components.Maui#27] [Ideas]
Dear yiszza,

We hope this message finds you well. We are the team behind [MAUIsland](https://github.com/Strypper/mauisland), a center place where people can come and display their .NET MAUI components to showcase to developers how to use them.

Here is a brief summary of our project
![image](https://github.com/mdc-maui/mdc-maui/assets/32596308/5ec368c1-fc26-482c-a872-7a69db821b4d)
![image](https://github.com/mdc-maui/mdc-maui/assets/32596308/be579553-320b-4cc6-b31b-f46e8bd06963)
![image](https://github.com/mdc-maui/mdc-maui/assets/32596308/0f2386a7-3431-4f0c-ab34-e209efae83f4)

We recently came across your collection of material UI components made for .NET MAUI on your GitHub repository and were very impressed by your work. We believe that your components would be a valuable addition to our project and would love to invite you to become a contributor to MAUIsland.

As a contributor, you would have the opportunity to define your components within MAUIsland and share your expertise with our community of developers. We believe that this collaboration would be mutually beneficial and would help to further the development of .NET MAUI.

Please let us know if this is something that interests you. We look forward to hearing from you and hope to have the opportunity to work together.

Best regards, 
Totechs team

**Comment by yiszza** [comment]
Thank you for inviting me, Mauisland is a amazing project!
But I’m afraid I don’t have enough time. I can’t participate in your amazing work until I finish the mdc-maui work.
Sorry!

**Comment by Strypper** [comment]
> Thank you for inviting me, Mauisland is a amazing project! But I’m afraid I don’t have enough time. I can’t participate in your amazing work until I finish the mdc-maui work. Sorry!

Thank you for considering our invitation and taking the time to look at our Mauisland project. We understand that you have other commitments and we appreciate your honesty. It’s unfortunate that you’re unable to participate at this time, but we hope to have the opportunity to collaborate with you in the future. Best of luck with your work on mdc-maui!

**Comment by yiszza** [comment]
Thank you!
